test/blocks/global-footer/mocks/base-footer.js

```diff
@@ -0,0 +1,3 @@
+
+export default `
+<footer class="global-footer" style="background-color: blue; height: 20vh"></footer>`;
```

Comment:
It would make sense to add some content to the footer, otherwise the checks if the elements are visible is not really appropriate, since you can't really see any rendered elements
![Screenshot 2023-10-19 at 17 04 45](https://github.com/robert-bogos/milo/assets/39759830/f5e56a6b-c8b4-44db-a9da-9f6518e6b834)